packages/databricks-vscode/src/bundle/BundleProjectManager.ts

```diff
@@ -126,9 +120,18 @@ export class BundleProjectManager {
             this.logger.debug("Project services have already been initialized");
             return;
         }
-        await this.configModel.init();
-        await this.connectionManager.init();
-        this.projectServicesReady = true;
+        const recordEvent = this.telemetry.start(
+            Events.EXTENSION_INITIALIZATION
+        );
+        try {
+            await this.configModel.init();
+            await this.connectionManager.init();
+            this.projectServicesReady = true;
+            recordEvent({success: true, type: "dabs"});
+        } catch (e) {
+            recordEvent({success: false, type: "dabs"});
+            throw e;
+        }
```

Comment:
This seems like a common pattern and something that can be abstracted and better wrapped in a try-catch-finally block. We can do this in a later PR.